gradle/libs.versions.toml

```diff
@@ -27,14 +27,15 @@ androidx-recyclerview = { module = "androidx.recyclerview:recyclerview", version
 bytebuddy-agent = { module = "net.bytebuddy:byte-buddy-agent", version.ref = "bytebuddy" }
 bytebuddy-core = { module = "net.bytebuddy:byte-buddy", version.ref = "bytebuddy" }
 
-compose-compiler = { module = "androidx.compose.compiler:compiler", version.ref = "composeCompiler" }
+compose-compiler = { module = "org.jetbrains.kotlin:kotlin-compose-compiler-plugin-embeddable", version.ref = "kotlin" }
```

Comment:
This should not be needed anymore. Apply the Compose plugin instead.

Reply:
I tried that, but paparazzi is a jvm module, so this line was failing when applying the Compose plugin:

```
add(AbstractKotlinPluginKt.PLUGIN_CLASSPATH_CONFIGURATION_NAME, libs.compose.compiler)
```

Reply:
Delete that line completely. The plugin will work fine in JVM modules.

Reply:
Ah, you're right.  I guess my issue here was related to the plugin test fixture configuration setup that I later fixed.  Good catch!